jena-fuseki2/jena-fuseki-main/src/main/java/org/apache/jena/fuseki/main/sys/FusekiModules.java

```diff
@@ -18,67 +18,56 @@
 
 package org.apache.jena.fuseki.main.sys;
 
-import java.util.ArrayList;
 import java.util.List;
 import java.util.function.Consumer;
 
-import org.apache.jena.base.module.Subsystem;
-
-/** Registry of modules */
+/**
+ * List of {@linkplain FusekiModule Fuseki modules}.
+ * This is the immutable unit of a modules for a server.
+ * <p>
+ * @see FusekiModulesLoaded
+ */
 public class FusekiModules {
 
-    private static Object lock = new Object();
+    /** A Fuseki module with no members. */
+    public static FusekiModules empty = FusekiModules.create();
```

Comment:
Make it `final` maybe? Doesn't seem to be changed in the class methods.

Reply:
Yes!